assignments/array-methods.js

```diff
@@ -55,29 +63,56 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
-let fullName = [];
+
+let fullName = runners.map((runner) =>{
+return { 'complete_name': `${runner['first_name']} ${runner['last_name']}`};
+});
+// runners.forEach(function(item){
+
+// })
 console.log(fullName);
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
-let allCaps = [];
-console.log(allCaps); 
+
+let allCaps = runners.map((runner) => {
+    return runner['first_name'].toUpperCase();
```

Comment:
It is best practices to use dot notation instead of square braces:
return runner.first_name.toUpperCase();